### PR TITLE
Fixed module component and routing

### PR DESCRIPTION
**Pull request by Lehren**

### CODE REVIEW

File: src/app/backend.service.ts [hunk 1]
```diff
@@ -16,8 +16,8 @@ export class BackendService {
   constructor(private http: HttpClient) {
   }
```
Comment (sjoerdbr) [inline src/app/backend.service.ts]: ONNODIGE SPACE

File: src/app/backend.service.ts [hunk 2]
```diff
@@ -16,8 +16,8 @@ export class BackendService {
...
 
-  getModuleContent(code: string): Observable<ModuleContent> {
-    const moduleContentUrl = 'http://172.17.0.1:8080/fmms/modules/' + code;
+  getModuleContent(curriculum: number, code: string): Observable<ModuleContent> {
+    const moduleContentUrl = 'http://172.17.0.1:8080/fmms/curriculum/' + curriculum + '/modules/' + code;
     return this.http.get<ModuleContent>(moduleContentUrl);
   }
```
Comment (sjoerdbr) [inline src/app/backend.service.ts]: ONNODIGE SPACE

File: src/app/components/module-component/module.component.ts [hunk 3]
```diff
@@ -13,16 +13,19 @@ export class ModuleComponent implements OnInit {
...
+     params => {
+       this.selectedModule = params['code'];
+       this.moduleCurriculum = params['curriculum'];
      });
   }
 
   ngOnInit(): void {
-    this.backendService.getModuleContent(this.selectedModule).subscribe(moduleContent => this.moduleContent = moduleContent);
+    this.backendService.getModuleContent(this.moduleCurriculum, this.selectedModule).subscribe(moduleContent => this.moduleContent = moduleContent);
   }
```
Comment (sjoerdbr) [inline src/app/components/module-component/module.component.ts]: DRIE MAAL ONNODIG SPACE

File: src/app/components/module-overview-component/module-overview.component.ts [hunk 4]
```diff
@@ -26,7 +26,7 @@ export class ModuleOverviewComponent implements AfterContentInit {
   }
 
   onClick(module_code: string): void {
-    this.router.navigate(['/module', module_code]);
+    this.router.navigate(['/curriculum/',this.selectedCurriculum.id,'modules', module_code]);
   }
```
Comment (sjoerdbr) [inline src/app/components/module-overview-component/module-overview.component.ts]: wat is al deze space

**Review by sjoerdbr — APPROVED** [review]

**Review by NNieuwenhuis — APPROVED** [review]

**Pull request merged**